tests/unit/test_indexapi_client_single.py

```diff
@@ -192,3 +196,57 @@ def test_update(monkeypatch):
         call("/index/testid", json={"description": "test"}),
         call("/index/testid", json={"description": "test", "status": exis_status}),
     ]
+
+
+@freeze_time(now := pd.Timestamp.now())
+@pytest.mark.parametrize("already_locked", [True, False])
+def test_lock(monkeypatch, already_locked):
```

Comment:
Did you intend to commit these? (not related to the current PR, so just making sure)

Reply:
yes, unrelated tests agree